tests/test_receiver.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3.5m
```

Comment:
Instead of having tests specifically for the plugins/ipc, what do you think of running all of the tests in the tests.py file I added with all of the modules? So we'd have unittest.main() being called several times, each with a setting specifying one of the modules. You need to change the run_vsm function in tests.py to use the input data in the appropriate fashion for each, of course.